pkg/datapath/iptables/iptables.go

```diff
@@ -1887,6 +1891,37 @@ func (m *Manager) addCiliumNoTrackEncryptionRules() (err error) {
 	return nil
 }
 
+func (m *Manager) addCiliumAcceptTunnelRules() (err error) {
+	port := m.sharedCfg.TunnelPort
+
+	if !m.sharedCfg.TunnelingEnabled || port == 0 {
+		return nil
+	}
+
+	cmd := []string{
+		"-t", "filter",
+		"-A", ciliumOutputChain,
+		"-p", "udp",
+		"--dport", strconv.Itoa(int(port)),
+		"-m", "comment", "--comment", "cilium: ACCEPT for tunnel traffic",
+		"-j", "ACCEPT",
+	}
+
+	if m.sharedCfg.EnableIPv4 {
+		if err := m.ip4tables.runProg(cmd); err != nil {
+			return err
+		}
+	}
+
+	if m.sharedCfg.EnableIPv6 {
+		if err := m.ip6tables.runProg(cmd); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
```

Comment:
(For a future PR) - looking at this, it feels like we should integrate the logic that installs `NOTRACK` and `ACCEPT` better. Checking the identical conditions twice is bound to drift over time.

What I'd expect from the overall codeflow is that it **first** install the `ACCEPT` rules, and only then the **identical** `NOTRACK` rules. Driven from the same conditions, using the same rule templates etc.